special-pages/pages/new-tab/app/components/App.js

```diff
@@ -43,16 +65,23 @@ export function App({ children }) {
                             />
                         )}
                     </CustomizerMenuPositionedFixed>
-                    {children}
-                </div>
-            </main>
-            {customizerKind === 'drawer' && (
-                <aside id={drawerId} class={styles.aside} aria-hidden={hidden}>
-                    <div class={styles.asideContent}>
-                        <CustomizerDrawer onClose={close} wrapperRef={wrapperRef} displayChildren={displayChildren} />
-                    </div>
-                </aside>
-            )}
-        </div>
+                </main>
+                {customizerKind === 'drawer' && (
+                    <aside
+                        class={cn(styles.aside, styles.asideScroller)}
+                        tabindex={tabIndex}
+                        aria-hidden={hidden}
+                        data-theme={browser}
+                        data-browser-panel
+                    >
+                        <div class={styles.asideContent}>
+                            <div class={styles.asideContentInner}>
```

Comment:
these extra divs were needed in the end to create a perfect scroll bar that doesn't move the content in the drawer

Reply:
(which will become clear in the next PR 😓 )